packages/opds-web-client/src/components/__tests__/Book-test.tsx

```diff
@@ -319,18 +251,19 @@ describe("Book", () => {
 
     it("shows summary, in book's language, with html stripped out and more link", () => {
       let context = mockRouterContext();
+      let store = buildStore();
       wrapper = mount(
-        <Book
-          book={book}
-          updateBook={updateBook}
-          fulfillBook={stub()}
-          indirectFulfillBook={stub()}
-        />,
+        <Provider store={store}>
+          <ActionsProvider>
```

Comment:
@kristojorg  is there an alternative to using the providers here for mounting a component to test? I can see this expanding to many providers later on and having it cleaner to look at and easier to test would be great.

Reply:
I would suggest we rely on standard react composition or functions to remove that boilderplate. Would something like this help? 
```js
const AppContext = ({store, children, ...etc}) => (
<Provider store={store}>
    <ActionsProvider>
        {children}
    </ActionsProvider>
</Provider>
```

You then just wrap with that when testing. Alternatively you could do something like
```js
const render = (store, ...etc) => (
<Provider store={store}>
    <ActionsProvider>
        {children}
    </ActionsProvider>
</Provider>
```
as a standard function that you call. 

I'll do a little more research though to see if there are better ways to do this

Reply:
The first one looks good. For now, I did what's in this PR: https://github.com/NYPL-Simplified/opds-web-client/pull/250/files#diff-f67235cabd8fcea381a5d0791f5947d9R29-R35

Reply:
Nice yea, I think that's basically the second one. Looks good to me